dll/win32/comdlg32/colordlg.c

```diff
@@ -373,7 +373,7 @@ static BOOL CC_MouseCheckColorGraph( HWND hDlg, int dlgitem, int *hori, int *ver
 }
 /***********************************************************************
  *                  CC_MouseCheckResultWindow                 [internal]
- *                  test if double click one of the result colors
+ *                  test if double-click one of the result colors
```

Comment:
This is 3rd-party code, please send the fix upstream.

https://git.reactos.org/?p=reactos.git;a=blob;f=media/doc/WINESYNC.txt;hb=50774f865c4f9d9b926f32a8bcfc20bbeced8fb5#l60